Sources/Public/CalendarViewContent.swift

```diff
@@ -173,6 +173,21 @@ public final class CalendarViewContent {
     return self
   }
 
+  /// Configures the accessibility focus override to be on a particular day after `CalendarView`'s content is set. If you'd like
+  /// VoiceOver to focus on a particular day (e.g. a day that your feature code considers to be selected), then invoke this function with a
+  /// date representing the target day.
+  ///
+  /// - Parameters:
+  ///   - date: The day to be focused, specified with a `Date` instance which will be used to determine the associated day
+  /// using the `calendar` instance with which `CalendarViewContent` was instantiated.
+  /// - Returns: A mutated `CalendarViewContent` instance with a new accessibility focus override.
+  public func withAccessibilityFocusOverride(onDayContaining date: Date?) -> CalendarViewContent {
```

Comment:
```suggestion
  public func withAccessibilityInitialFocusOverride(onDayContaining date: Date?) -> CalendarViewContent {
```
Might be helpful to use the term `initial` here